ovos_plugin_common_play/ocp/utils.py

```diff
@@ -1,9 +1,27 @@
+import mimetypes
 import shutil
 import tempfile
 from os import makedirs
 from os.path import basename, expanduser, isfile, join, dirname
 
 from ovos_plugin_common_play.ocp.status import TrackState, PlaybackType
+from ovos_plugin_manager.ocp import StreamHandler
+
+ocp_plugins = StreamHandler()
+
+
+def find_mime(uri):
+    """ Determine mime type. """
+    mime = mimetypes.guess_type(uri)
+    if mime:
+        return mime
+    else:
+        return None
+
+
+def available_extractors():
+    return ["/", "http"] + \
+           [f"{sei}//" for sei in ocp_plugins.supported_seis]
```

Comment:
should this be `{sei}://`? Not sure how network or local file paths expand without the `:`

Reply:
skill in the wild are already using the old scheme, so this is not so trivial to change. but this is all stripped by OCP itself before playback so not sure what you mean by expand?

Reply:
i think there was an issue somewhere in the pipeline splitting at `:` and thats why i didnt add it originally, but i don't recall details....